client/state/editor/reducer.js

```diff
@@ -75,6 +75,17 @@ export function isIframeLoaded( state = false, action ) {
 	return state;
 }
 
+export function iframePort( state = null, action ) {
```

Comment:
I think this reducer could use a couple of tests 😉

Reply:
done with b252597cac